tools/provision/provision.go

```diff
@@ -205,6 +209,23 @@ func Provision(conf Config) {
 		}
 		fmt.Println("")
 	}
+
+	var wg sync.WaitGroup
+	fmt.Println("# list of channels that things can publish to")
+	fmt.Println("# each channel is connected to each thing from things list")
+	for i := 0; i < conf.Num; i++ {
+		for j := 0; j < conf.Num; j++ {
+			wg.Add(1)
+			go func(wg *sync.WaitGroup, i, j int) {
+				defer wg.Done()
+				s.ConnectThing(things[j].ID, *channels[i], token)
+			}(&wg, i, j)
+		}
+		fmt.Println("[[channels]]")
```

Comment:
A single `fmt.Printf(...)` instead of multiple `fmt.Println(...)`?

Reply:
Agreed